projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/sensitivity/normalstirfutures/NormalSTIRFuturesSensitivityFDCalculator.java

```diff
@@ -83,30 +97,32 @@ public MultipleCurrencyParameterSensitivity calculateSensitivity(final Instrumen
         final double[] yieldBumpedPlus = curveInt.getYDataAsPrimitive().clone();
         yieldBumpedPlus[loopnode] += _shift;
         final YieldAndDiscountCurve dscBumpedPlus = new YieldCurve(curveInt.getName(), new InterpolatedDoublesCurve(curveInt.getXDataAsPrimitive(), yieldBumpedPlus, curveInt.getInterpolator(), true));
-        final NormalSTIRFuturesSmileProviderDiscount marketDscBumpedPlus = new NormalSTIRFuturesSmileProviderDiscount(black.getMulticurveProvider().withDiscountFactor(ccy, dscBumpedPlus),
-            black.getNormalParameters(), black.getFuturesIndex());
+        final NormalSTIRFuturesExpStrikeProviderDiscount marketDscBumpedPlus = new NormalSTIRFuturesExpStrikeProviderDiscount(
```

Comment:
The method accept those types: NormalSTIRFuturesExpStrikeProviderDiscount and NormalSTIRFuturesExpSimpleMoneynessProviderDiscount, but in the bump seems to create only a NormalSTIRFuturesExpStrikeProviderDiscount.